### PR TITLE
Check if is in gold mode to call make task gol api when user click su…

### PR DESCRIPTION
**Pull request by mariarivera30**
…bmit

### CODE REVIEW

File: pybossa.js [hunk 1]
```diff
@@ -54,13 +54,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
     function _saveTaskRun(taskrun) {
-        return $.ajax({
-            type: 'POST',
-            url: url + 'api/taskrun',
-            dataType: 'json',
-            contentType: 'application/json',
-            data: taskrun
-        });
+        if(window.pybossa.isGoldMode){
+            const projectId = JSON.parse(taskrun).project_id;
```
Comment (fiorda) [inline pybossa.js]: maybe you can pass the project id from the `_createTaskRun` function

Comment (fiorda) [reply pybossa.js]: do you think is ok to use `const` here?

File: pybossa.js [hunk 2]
```diff
@@ -54,13 +54,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+            return $.ajax({
+                type: 'POST',
+                url: url + `api/project/${projectId}/taskgold`,
```
Comment (fiorda) [inline pybossa.js]: Looks like the only thing that changes between the requests is the url, maybe the if could just set that

Comment (fiorda) [reply pybossa.js]: String template might not work on all browsers

Comment (mariarivera30) [reply pybossa.js]: agree

File: pybossa.js (hunk 1 above)
Comment (mariarivera30) [reply pybossa.js]: agree

**Pull request merged**